### PR TITLE
Add web crypto demo

### PR DESCRIPTION
**Pull request by bencmbrook**
Adds a comparison vs. Web Crypto.

Adds an animation to show any differences in browser responsiveness/UI jank.

Resolves: https://github.com/transcend-io/penumbra/issues/125

### CODE REVIEW

File: example/web-crypto.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    const decryptedUrl = URL.createObjectURL(decryptedBlob);
+    return { type: 'uri', data: decryptedUrl, mimetype };
+  });
+
+  try {
+    insertIntoCell(promises);
+  } catch (err) {
+    console.error(err);
+  }
+};
```
Comment (bencmbrook) [inline example/web-crypto.js:252]: This is the "implementation". The rest of this PR is visual/extraneous

Comment (eligrey) [reply example/web-crypto.js:252]: Thanks, this is very readable!

**Comment by bencmbrook** [comment]
@eligrey added an animation to the demo to compare any UI jank. Check out the Netlify deploy preview

**Comment by eligrey** [comment]
Wow, this is a nice visualization of main-thread jank 👍. I can clearly see a ton of jank in the main-thread WebCrypto demo, with our web worker implementation being immensely more responsive!

**Pull request merged**